src/ngraph/frontend/onnx_import/attribute.hpp

```diff
@@ -16,6 +16,7 @@
 
 #pragma once
 
+#include <onnx.pb.h>
```

Comment:
Should this be `"onnx.pb.h"` instead of `<onnx.pb.h>`

Reply:
Fixed.